services/QuillLMS/spec/factories/activities.rb

```diff
@@ -87,7 +87,7 @@
       end
     end
 
-    factory :evidence_activity do
+    factory :evidence_lms_activity do
```

Comment:
I believe this is now a redundant factory. In a future PR we should dedupe and clean this up, so that specs only use `evidence_activity`, not `evidence_lms_activity`

Reply:
I also, did a double take on this.  I think the `evidence_lms_activity` is the Activity model in the LMS whereas evidence_activity is Evidence::Activity model.

While this seems correct, I'm wondering if there's better naming convention to avoid this convention in the future.